v2/cn/deb-packages.md

```diff
@@ -0,0 +1,187 @@
+<!---
+    @title         Deb Packages
+--->
+
```

Comment:
I think @moonming has already done this page's CN translation here:

https://github.com/openresty/openresty.org/blob/master/v2/cn/deb-packages.html